nodestream_akamai/akamai_utils/model.py

```diff
@@ -39,7 +39,7 @@ def origin_count(self):
     def hostname_count(self):
         return len(self.hostnames)
 
-    def as_eventbus_json(self):
+    def dict_factory(self):
```

Comment:
For data classes, this is a magic named function that allows `dataclasses.asdict()` to provide non-standard/custom results.